assets/model_monitoring/components/generation_safety_quality/generation_safety_quality_signal_monitor/spec.yaml

```diff
@@ -134,6 +134,8 @@ jobs:
         type: mltable
       similarity_violations:
         type: mltable
+      evaluation:
```

Comment:
can you make this evaluation as a signal output? i.e. adding another port here https://github.com/Azure/azureml-assets/blob/ed10abd8d33d80a5e447d94872a00cfad6374763/assets/model_monitoring/components/generation_safety_quality/generation_safety_quality_signal_monitor/spec.yaml#L97C1-L98C1

I think this does not require extra service change.

Reply:
on making a separate signal output - in that case, I would need to have GSQ pipeline output a second output port.  I think I might need to add another Metric Outputter component in that case:
![image](https://github.com/Azure/azureml-assets/assets/24683184/79526127-ef28-487f-aac9-3eed92f24330)

Reply:
so this seems like a separate feature/PR in and of itself, but I think we should talk to Corey about that first